bin/ch/Helpers.cpp

```diff
@@ -133,9 +58,9 @@ HRESULT Helpers::LoadScriptFromFile(LPCSTR filename, LPCWSTR& contents, bool* is
     //
     // Read the entire content as a binary block.
     //
-    fread((void*)contentsRaw, sizeof(char), lengthBytes, file);
+    fread(pRawBytes, sizeof(BYTE), lengthBytes, file);
     fclose(file);
-    *(WCHAR*)((byte*)contentsRaw + lengthBytes) = _u('\0'); // Null terminate it. Could be LPCWSTR.
+    *reinterpret_cast<WCHAR*>(pRawBytes + lengthBytes) = 0; // Null terminate it. Could be UTF16
```

Comment:
Didn't understand these, could you elaborate more? 1) This doesn't depend on how many bytes char16_t is (however we only support up to WCHAR - 2 bytes). 2) What behaves differently, and why that matters? 3) Never heard of it, why is reinterpret_cast for read only?

I changed a variable name and cast style, otherwise this is same as old code. The purpose is to null terminate it no matter the loaded source is utf8 or utf16 (and we only support utf16 LE).